qemu/tests/qmp_basic.py

```diff
@@ -332,7 +335,10 @@ def argument_checker_suite(monitor):
 
         # value argument must be a json-number
         for arg in ({}, [], True, "foo"):
-            resp = monitor.cmd_qmp("migrate_set_speed", {"value": arg})
+            if utils_misc.compare_qemu_version(5, 1, 0) is True:
+                resp = monitor.cmd_qmp("migrate-set-parameters", {"value": arg})
```

Comment:
To replace `migrate-set-speed` with `migrate-set-parameters`, parameter name `max-bandwidth` is must:
`resp = monitor.cmd_qmp("migrate-set-parameters", {"max-bandwidth": arg})`

However,  a simple replacement can not match the test requirement as my understanding,  the purpose is: `value argument must bz a json number`,  the new `migrate-set-parameter` can not cover this scenario.  To cover this scenario, i think we need find another cmd.

Reply:
> To replace `migrate-set-speed` with `migrate-set-parameters`, parameter name `max-bandwidth` is must: `resp = monitor.cmd_qmp("migrate-set-parameters", {"max-bandwidth": arg})`

I got it. but source code has errors, the value argument of migrate_set_speed  is a json-int and not a json-number.

> However, a simple replacement can not match the test requirement as my understanding, the purpose is: `value argument must bz a json number`, the new `migrate-set-parameter` can not cover this scenario. To cover this scenario, i think we need find another cmd.

1. migrate-set-parameter can cover this scenario
2. the value argument of downtime-limit is a json-number.
        for arg in ({}, [], True, "foo"):
            if utils_misc.compare_qemu_version(5, 1, 0) is True:
                resp = monitor.cmd_qmp("migrate-set-parameters", {"downtime-limit": arg})
                check_error_resp(resp, "GenericError",
                                 {"name": "downtime-limit", "expected": "number"})
            else:
                resp = monitor.cmd_qmp("migrate_set_downtime", {"value": arg})
                check_error_resp(resp, "GenericError",
                                 {"name": "value", "expected": "number"})

Reply:
According to your test result,  It is ok for me to replace it with `donwtime-limt`